css/_cardlist.scss

```diff
@@ -0,0 +1,76 @@
+// Card List
+//
+//	The Teleopti Card List is based on the card list displayed in google inbox.  Each card can be clicked on within the list and will expand to show detail.</br>
+//  Only one card can be opened at a time.The header and content of each card is completely customizable and any html elements provided within the &lt;card-header&gt; and  &lt;card-body&gt; tags will be rendered appropriately.
+//
+//Markup:
+//<div id="myChart"></div>
+//<teleopti-card-list>
+//	<teleopti-card ng-repeat="item in items">
+//		<card-header>
+//			<span><i class="mdi" ng-class="item.title"></i></span>
+//			<span style="float: right;"><i class="mdi mdi-chevron-right"></i></span>
+//		</card-header>
+//		<card-body>
+//			<button class="wfm-fab info material-depth-1"><i class="mdi mdi-duck"><md-tooltip>Ducks</md-tooltip></i></button>
+//		</card-body>
+//	</teleopti-card>
+//</teleopti-card-list>
+//
+//Styleguide 2.4
+//
+
+
+/*Overrides bootstrap*/
+.teleopti-card-list {
+	width: 100%;
+}
+
+.teleopti-card-list-container {
+	max-height: 450px;
+}
+
+
+md-card {
```

Comment:
no no ^^ it's just the lines 34 and 35 : 
md-card {
}
there is no css inside the declaration.